OpenQA/Isotovideo/Utils.pm

```diff
@@ -200,7 +200,7 @@ sub load_test_schedule {
         unshift @INC, '.' unless path($bmwqemu::vars{CASEDIR})->is_abs;
         bmwqemu::fctinfo 'Enforced test schedule by \'SCHEDULE\' variable in action';
         $bmwqemu::vars{INCLUDE_MODULES} = undef;
-        autotest::loadtest($_ =~ qr/\./ ? $_ : $_ . '.pm') foreach split(',', $bmwqemu::vars{SCHEDULE});
+        autotest::loadtest($_ =~ qr/\./ ? $_ : $_ . '.pm') foreach split(/,|\s+/, $bmwqemu::vars{SCHEDULE});
```

Comment:
Maybe it would be enough to allow a literal space, but not all kinds of whitespace `\s`

Reply:
I thought about only space here as well, but I do not see a problem with `\s` and somehow I liked the idea to allow newlines or tabs as well.


> I'm personally fine with it. Spaces shouldn't be used in filenames here anyways, I suppose.


I checked `os-autoinst-dist-opensuse` with `find . | grep -P '\s'` there was nothing..

Reply:
@perlpunk  do you think we should go with space only? Like this:
```suggestion
        autotest::loadtest($_ =~ qr/\./ ? $_ : $_ . '.pm') foreach split(/,| +/, $bmwqemu::vars{SCHEDULE});
```
or you ok with `\s` ?